src/odemis/util/test/util_test.py

```diff
@@ -396,54 +401,255 @@ def test_line_clipping(self):
+                                                                "the closest index to 5 in %s" % in_exp)
+
+
+class NormalizeRectTestCase(unittest.TestCase):
+
+    def test_already_normalized(self):
+        """ Test normalizing a rectangle with already normalized values """
+        rect = (1, 2, 3, 4)
+        normalized_rect = util.normalize_rect(rect)
+        self.assertEqual(normalized_rect, (1, 2, 3, 4), "Failed to normalize the rectangle")
+
+    def test_non_normalized(self):
+        """ Test normalizing a rectangle with not yet normalized values """
+        rect = (10, 7, 5, 3)
+        normalized_rect = util.normalize_rect(rect)
+        self.assertEqual(normalized_rect, (5, 3, 10, 7), "Failed to normalize the rectangle")
+
+
+class IsPointInRectTestCase(unittest.TestCase):
+
+    def test_points_in_rect(self):
+        """ Test whether the given coordinates are in the rectangle """
+        rect = (5, 3, 10, 7)
+        point = (8.0, 4.0)
+        in_rect = util.is_point_in_rect(point, rect)
+        self.assertTrue(in_rect, "Failed to get correct output for the point in rectangle %s with "
+                                 "point %s" % (rect, point))
+
+    def test_points_outside_rect(self):
+        """ Test whether the given coordinates are not in the rectangle """
+        rect = (5, 3, 10, 7)
+        point = (22.0, 44.0)
+        in_rect = util.is_point_in_rect(point, rect)
+        self.assertFalse(in_rect, "Failed to get correct output for the point in rectangle %s with "
+                                  "point %s" % (rect, point))
+
+
+class ExpandRectTestCase(unittest.TestCase):
+
+    def test_expand_rect(self):
+        """ Test expanding a rectangle """
+        rect = (5, 3, 10, 7)
+        expanded_rect = util.expand_rect(util.normalize_rect(rect), 5)
+        expected_rect = (0, -2, 15, 12)
+        self.assertEqual(expanded_rect, expected_rect,  "Failed to get correct output for rectangle %s with "
+                                                        "expanded rectangle %s" % (expanded_rect, expected_rect))
+
+
+class FindPlotContentTestCase(unittest.TestCase):
+
+    def test_find_plot_content(self):
+        """ Test finding left and right most non-zero values"""
+        xd = (10, 15, 20, 25, 30)
+        yd = (0, 0, 1, -2, 0)
+        plot_content = util.find_plot_content(xd, yd)
+        self.assertEqual(plot_content, (20, 25), "Failed to get correct output while finding plot content "
+                                                 "for %s with values  %s %s" % (plot_content, xd, yd))
+
+
+class WrapToMpiPpiTestCase(unittest.TestCase):
+
+    def test_positive_pi(self):
+        """ Test value is smaller than +pi value """
+        converted_angle = util.wrap_to_mpi_ppi(4.65)
+        self.assertLessEqual(converted_angle, math.pi, "Failed to convert the angle to a value between -pi and +pi "
+                                                  "with the converted value: %s" % converted_angle)
```

Comment:
I'm not sure if it's a PEP8 rule or not, but it looks a bit nicer if you align the string with the string above it:
```
        self.assertLessEqual(converted_angle, math.pi, "Failed to convert the angle to a value between -pi and +pi "
                                                       "with the converted value: %s" % converted_angle)
```